<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenGL Demo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>A:/PNG/png/IDisk Globe.png</normaloff>A:/PNG/png/IDisk Globe.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color:rgb(68,68,68);
	color:rrgb(255,255,255);
	font: 10pt &quot;幼圆&quot;
}

QMenuBar{
	background-color:rgb(200,200,200);
	color:rgb(60,60,60);
}
QWidget:checked{background-color:rgb(150, 150, 150);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="DHOpenGLWidget" name="openGLWidget">
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>800</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>201</width>
          <height>800</height>
         </size>
        </property>
        <property name="title">
         <string>控制界面</string>
        </property>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>181</width>
           <height>154</height>
          </rect>
         </property>
         <property name="title">
          <string>对象姿态角</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Yaw</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSlider" name="hSlider_ObjYaw">
            <property name="maximum">
             <number>360000</number>
            </property>
            <property name="value">
             <number>45000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Pitch</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QSlider" name="hSlider_ObjPitch">
            <property name="maximum">
             <number>360000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Roll</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QSlider" name="hSlider_ObjRoll">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <number>360000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>280</y>
           <width>181</width>
           <height>154</height>
          </rect>
         </property>
         <property name="title">
          <string>相机姿态角</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Yaw</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSlider" name="hSlider_CamYaw">
            <property name="maximum">
             <number>360000</number>
            </property>
            <property name="value">
             <number>180000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Pitch</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QSlider" name="hSlider_CamPitch">
            <property name="maximum">
             <number>360000</number>
            </property>
            <property name="value">
             <number>315000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Roll</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QSlider" name="hSlider_CamRoll">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <number>360000</number>
            </property>
            <property name="value">
             <number>180000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>450</y>
           <width>181</width>
           <height>154</height>
          </rect>
         </property>
         <property name="title">
          <string>相机中心点</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSlider" name="hSlider_CamPosX">
            <property name="minimum">
             <number>-15</number>
            </property>
            <property name="maximum">
             <number>15</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QSlider" name="hSlider_CamPosY">
            <property name="minimum">
             <number>-15</number>
            </property>
            <property name="maximum">
             <number>15</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Z</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QSlider" name="hSlider_CamPosZ">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <number>15</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QCheckBox" name="cbx_worldAxis">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>131</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>世界坐标轴</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>查看</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actClear"/>
   <addaction name="actDraw_Rect"/>
   <addaction name="actWireframe"/>
   <addaction name="actionLinearFilter"/>
  </widget>
  <action name="actClear">
   <property name="text">
    <string>清空图形</string>
   </property>
   <property name="toolTip">
    <string>清空图形</string>
   </property>
  </action>
  <action name="actDraw_Rect">
   <property name="text">
    <string>绘制图形</string>
   </property>
   <property name="toolTip">
    <string>绘制图形</string>
   </property>
  </action>
  <action name="actWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>线框模式</string>
   </property>
  </action>
  <action name="actionLinearFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>线性材质过滤</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DHOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">DHOpenGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
